src/Components/Charts/BentoPie.tsx

```diff
@@ -49,13 +51,16 @@ const BentoPie = ({
   onClick,
   sort = true,
   colorTheme = 'default',
-  chartThreshold = useChartThreshold(),
-  maxLabelChars = useChartMaxLabelChars(),
+  chartThreshold,
+  maxLabelChars,
   ...params
 }: PieChartProps) => {
   const t = useChartTranslation();
   const { fill: theme } = useChartTheme().pie[colorTheme];
 
+  const resolvedChartThreshold = useResolvedValue(chartThreshold, useChartThreshold);
```

Comment:
i don't think passing a hook as an argument is recommended.

Reply:
I guess I over engineered it